### ISSUE TITLE
Show tabs from all windows

### ISSUE THREAD

**Issue by gitlover4ever** [josh-berry/tab-stash#314]
### Problem Statement

**Tab Stash only shows tabs from the current window.**

This is especially a problem when having multiple displays, as two windows are almost always open. This means you can't stash an entire session, without manually opening tab stash in both windows, stashing and then combining the groups.

### Preferred Solution(s)

Open tabs should show all open tabs in all windows. They could even be sorted by windows. If other people want to only show open tabs of the current window, an option to enable/disable could be implemented.

**Comment by josh-berry** [comment]
I agree this would be useful, I'll definitely keep it in mind! Thanks for the feature request!

**Comment by ddddavidee** [comment]
This, plus creating and managing different "stash-es" from the extension homepage would make this extension the best "tab group manager" ever.

**Comment by mindthump** [comment]
this might be the subject of a separate feature request, but I'd like to be able to stash all open tabs from all windows at once. (OneTab has a similar action.) This is very helpful when you want to reboot or just quit the app and be sure everything is saved.

**Comment by josh-berry** [comment]
Just making a note that per #346, being able to open selected tabs in a new window would be useful as part of this feature.

**Comment by joeytwiddle** [comment]
Regarding this feature, I'm concerned that having all windows at the top of the list might obscure the stash groups below them. (Imagine that I have 5 windows, and some of them have many tabs in them.)

What might help with that: If I could click to collapse all of the windows at once. Or, if all windows except the current one start off already collapsed.

But these are probably concerns for later. Just something to bear in mind.

**Comment by Nezteb** [comment]
I've checked the docs for Firefox itself and tried several extensions that manage open tabs as bookmarks (the latest being Tab Stash), but I've never been able to find a way to "bookmark all open tabs in all windows".

One extension I am aware of that can operate on content in tabs from all windows is "Search all Tabs".

- The extension: https://addons.mozilla.org/en-US/firefox/addon/search-all-tabs/
- The code that seems to be grabbing all tabs in all windows: https://github.com/lunu-bounir/search-all-tabs/blob/master/v3/data/popup/index.js#L143-L160

I'm relatively unfamiliar with browser extension development but if I find any free time I'll try to dig in and open a draft PR. 😅
